abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+      },
+      {
+        "title": "Verify SSL",
+        "description": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is set to True.",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_threats",
+      "annotation": "get_threats",
+      "category": "investigation",
+      "title": "Get Threats",
+      "description": "Retrieves detailed list of threats based on the parameters that you have specified",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Filter",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "filter",
+          "tooltip": "Specify the filter, currently the only key that is supported is receivedTime. At least 1 of gte/lte must be specified, with a datetime string following the YYYY-MM-DDTHH:MM:SSZ format.",
+          "description": "Specify the filter, currently the only key that is supported is receivedTime. At least 1 of gte/lte must be specified, with a datetime string following the YYYY-MM-DDTHH:MM:SSZ format.",
+          "placeholder": "receivedTime gte 2020-01-01T01:01:01Z lte 2021-12-01T01:01:01Z"
+        },
+        {
+          "title": "Page Size",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "integer",
+          "name": "pageSize",
+          "tooltip": "Specify the number of threats on each page. Each page of data will have at most pageSize threats. Has no effect if filter is not specified. Default value is 100.",
+          "description": "Specify the number of threats on each page. Each page of data will have at most pageSize threats. Has no effect if filter is not specified. Default value is 100."
+        },
+        {
+          "title": "Page Number",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "integer",
+          "name": "pageNumber",
+          "tooltip": "1-indexed page number to get a particular page of threats. Has no effect if filter is not specified. Default value is 1.",
+          "description": "1-indexed page number to get a particular page of threats. Has no effect if filter is not specified. Default value is 1."
+        },
+        {
+          "title": "Source",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "name": "source",
+          "tooltip": "Filters threats based on the source of detection. Default value is all.",
+          "description": "Filters threats based on the source of detection. Default value is all.",
+          "options": [
+            "all",
```

Comment:
Make the options more friendly looking for UI.

Reply:
done